src/commands/pwd_command.c

```diff
@@ -1,13 +1,29 @@
 #include <libft.h>
+#include <env/environment.h>
 #include <commands/commands.h>
 
+char	*get_pwd(char *buffer)
+{
+	const t_env	*pwd = find_variable(get_environment(), "PWD");
+
+	getcwd(buffer, BUFFER_SIZE);
+	if (!buffer && !pwd)
+		return (NULL);
+	else if (!buffer && pwd && pwd->value)
+		buffer = pwd->value;
```

Comment:
instead of doing this, could you do strlcpy?
we're receiving a char *buffer[], don't think it is correct to just make it point to a string that is allocated.